### PR TITLE
Use instance name instead of app name in config

### PR DESCRIPTION
**Pull request by Aeris1One**
## Problem
- #92 

## Solution
- *Simply use `app=$YNH_APP_INSTANCE_NAME`*

## PR Status
- [x] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/etherpad_mypads_ynh%20PR93/)  
When the PR is marked as ready to merge, we have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/config [hunk 1]
```diff
@@ -15,7 +15,7 @@ source _variables
 # RETRIEVE ARGUMENTS
 #=================================================
 
-app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
+app=$YNH_APP_INSTANCE_NAME
```
Comment (maniackcrudelis) [inline scripts/config:18]: That line is actually, use $YNH_APP_INSTANCE_NAME, and if it's empty, use $YNH_APP_ID instead.
If you end up with $YNH_APP_ID, then you should check that PR to see if you actually got anything in $app.

**Comment by maniackcrudelis** [comment]
I actually just tried with YunoHost 3.7 and the testing branch.
I got this:
```
app=etherpad_mypads__2
YNH_APP_ID=etherpad_mypads
YNH_APP_INSTANCE_NAME=etherpad_mypads__2
```
Everything looks good

**Comment by Aeris1One** [comment]
When changing anything in config panel (testing branch) for etherpad_mypad__3.
```
Nous n’avons pas trouvé l’application « etherpad_mypads » dans la liste des applications installées:
* etherpad_mypads__2
* etherpad_mypads__3
```

**Comment by maniackcrudelis** [comment]
Well ok, that's strange...
Could you edit that file, `/etc/yunohost/apps/etherpad_mypads__3/scripts/config` and add some debug
```
ynh_debug -m "YNH_APP_INSTANCE_NAME=$YNH_APP_INSTANCE_NAME"
ynh_debug -m "YNH_APP_ID=$YNH_APP_ID"
ynh_debug -m "app=$app"
```
Just after the line `app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}`

**Comment by maniackcrudelis** [comment]
> Nous n’avons pas trouvé l’application « etherpad_mypads » dans la liste des applications installées:

I'm not sure actually that's a problem with the script... We'll see what the debug says, but I feel like that a core issue. Before the script is even started.

**Comment by maniackcrudelis** [comment]
Indeed, a core issue !
With etherpad_mypads and etherpad_mypads__2, that works, but probably the change are made on the first one ? Maybe not...

Anyway, with etherpad_mypads__2 and etherpad_mypads__3 and the first etherpad_mypads removed, I get the same error that you had.

That's not a problem with the script itself, that a problem with the core that doesn't handle correctly multi-instance for that feature.

**Comment by Aeris1One** [comment]
Will not merge

**Pull request closed without merging**